### DISCUSSION TITLE
Dá pra conseguir entrar no mercado de trabalho sem diploma?

### DISCUSSION THREAD

**Discussion by pauloviniciusdev** [frontendbr/forum#2121] [Discussão]
Olá! Me chamo Paulo e eu queria saber se dá pra conseguir um emprego sem diploma em ti. Eu sei... Eu sei... existem 1001 vídeos no Youtube falando que dá pra conseguir sem diploma, mas sempre que ouço isso acabo pensando em apenas uma coisa: NETWORKING! Sempre acho que o motivo de pessoas serem contratas sem diploma, é por causa de networking ou elas eram muito fodonas em programação.
Então eu queria perguntar aos bravos usuários do GitHub. Eai? o que vocês acham?

**Comment by wilsonneto-swe** [comment]
Com certeza, e mesmo sem networking, basta aplicar para as vagas, se você mesmo sem diploma ir bem no teste, a vaga é tua.
Atualmente eu participo do processo de seleção na empresa atual, na avaliação técnica, e acho que apenas uns 50% do pessoal ali tem diploma, na verdade eu foco no teste técnico e na experiencia da pessoa.

**Comment by expalmer** [comment]
dá

**Comment by ninetails** [comment]
Sim, é possível.

Tem empresa ou outra que pede diploma (ou que esteja pelo menos cursando)... Mercado Livre é uma delas pelo que eu saiba. E tem as governamentais como Serpro etc

Mas lembre do seguinte: se você não tem diploma e está começando, você já está começando sem um "diferencial". Networking é bom, mas as empresas sempre querem é contratar os melhores dado o escopo/budget disponível.

Exemplo: uma empresa tem uma vaga de Junior e no final a decisão ficou entre dois perfis, ambos pedindo um patamar salarial semelhante, ambos sem experiência e início imediato. Nisso começariam as comparações. Algum dos dois está cursando faculdade? Ou já terminou alguma? Se ambos não fizeram, quais cursos fizeram....

Networking irá te dar mais chances de ser contratado e mais cedo pois você terá mais oportunidades. Não necessariamente isso reflete em o que a pessoa tem de conhecimento. Pode até dizer o quão esforçado ou engajado que você é ao por exemplo, este networking ser fruto de vc frequentar meetups e eventos de tech e até pode ser um diferencial, mas não necessariamente seria uma correlação. Encare mais como um complemento.

Pessoas que são contratadas sem diploma geralmente são aqueles que já possuem diferencial, ou uma boa indicação (pois precisa ser muito amigo ou familiar de alguém pra te indicarem a uma vaga sem saber da tua experiência).

**Comment by romulosardinha** [comment]
Sim dá.

Na empresa em que estou ninguém foi escolhido por formação, mas por mostrar competência nas tecnologias que precisamos.

Porém imagino que para empresas mais tradicionais seria importante alguma formação específica.
